### PR TITLE
fixes bodybags and stasis bags not working with the medevac stretcher 

### LINKED ISSUE #4329: Medivac+stasis won't work
## Reproduction:
Try to Medivac a person inside a stasis bag, it will do the send up sound and message, but won't send the person or bag up.

### PR DESCRIPTION
**Pull request by MacBlaze1**
## About The Pull Request

This PR fixes the bug in which stasis bags do not work with the medevac stretcher 
Closes #4329 
## Why It's Good For The Game

Fixes good bugs bad

## Changelog
:cl:

fix: Corpsmen can now use stasis bags again with medevac stretchers 

/:cl:

### CODE REVIEW

File: code/game/objects/structures/stool_bed_chair_nest/bed.dm [hunk 1]
```diff
@@ -457,6 +456,7 @@ GLOBAL_LIST_EMPTY(activated_medevac_stretchers)
 	if(buckled_bodybag)
 		unbuckle_bodybag()
 		buckled_bodybag.forceMove(get_turf(linked_beacon))
+		buckled_bodybag = null
```
Comment (Rohesie) [inline code/game/objects/structures/stool_bed_chair_nest/bed.dm:459]:
```suggestion
		var/obj/structure/closet/bodybag/teleported_bodybag = buckled_bodybag
		unbuckle_bodybag()
		teleported_bodybag.forceMove(get_turf(linked_beacon))
```

File: code/game/objects/structures/stool_bed_chair_nest/bed.dm [hunk 2]
```diff
@@ -86,7 +86,6 @@ obj/structure/bed/Destroy()
 	buckled_bodybag.roller_buckled = null
 	buckled_bodybag.glide_modifier_flags &= ~GLIDE_MOD_BUCKLED
 	buckled_bodybag.reset_glide_size()
-	buckled_bodybag = null
```
Comment (Rohesie) [inline code/game/objects/structures/stool_bed_chair_nest/bed.dm:89]: Revert this change

File: code/game/objects/structures/stool_bed_chair_nest/bed.dm [hunk 3]
```diff
@@ -86,7 +86,6 @@ obj/structure/bed/Destroy()
 	buckled_bodybag.roller_buckled = null
 	buckled_bodybag.glide_modifier_flags &= ~GLIDE_MOD_BUCKLED
 	buckled_bodybag.reset_glide_size()
```
Comment (Rohesie) [inline code/game/objects/structures/stool_bed_chair_nest/bed.dm:88]:
```suggestion
	buckled_bodybag.reset_glide_size()
	buckled_bodybag = null
```

**Comment by spookydonut** [comment]
Don't bags also snap to display underneath medevac stretchers?
